src/transaction/log_recovery.c

```diff
@@ -1014,15 +1016,19 @@ log_recovery_redo (THREAD_ENTRY * thread_p, log_recovery_context & context)
     assert (log_recovery_redo_parallel_count >= 0);
     if (log_recovery_redo_parallel_count > 0)
       {
-	reusable_jobs.initialize (cublog::PARALLEL_REDO_REUSABLE_JOBS_COUNT, log_recovery_redo_parallel_count,
-				  cublog::PARALLEL_REDO_REUSABLE_JOBS_FLUSH_BACK_COUNT);
-	parallel_recovery_redo.reset (new cublog::redo_parallel (log_recovery_redo_parallel_count, nullptr,
+	reusable_jobs.initialize (log_recovery_redo_parallel_count);
+	parallel_recovery_redo.reset (new cublog::redo_parallel (log_recovery_redo_parallel_count, false,
 								 redo_context));
       }
   }
 #endif
 
-  const auto time_start = std::chrono::system_clock::now ();
+  /* 'setting_up' measures the time taken to initialize parallel log recovery redo infrstructure */
+  // *INDENT-OFF*
+  const auto time_dur_setting_up_ms = std::chrono::duration_cast <std::chrono::milliseconds>(
+        std::chrono::system_clock::now () - time_start_setting_up);
+  // *INDENT-ON*
+  const auto time_start_main = std::chrono::system_clock::now ();
```

Comment:
- A single `std::chrono::system_clock::now ()` call, under disabled indentation
- Avoid breaking after parenthesis if possible
- Nitpicking: remove `time_` from `time_dur_` to shorten the name.
```suggestion
  // *INDENT-OFF*
  const auto time_start_main = std::chrono::system_clock::now ();
  const auto dur_setting_up_ms =
    std::chrono::duration_cast <std::chrono::milliseconds> (time_start_main - time_start_setting_up);
  // *INDENT-ON*
```